packages/Sandblocks-Core/SBEditor.class.st

```diff
@@ -90,6 +90,14 @@ SBEditor class >> openFor: aMethodOrClass [
 	^ (self newFor: aMethodOrClass) openInWindow; yourself
 ]
 
+{ #category : #generated }
```

Comment:
If you change it `*generated` it won't be tracked in git, which is probably what you want in this case?